### PR TITLE
Add active_volunteer_count to summary endpoint

### LINKED ISSUE #198: Add `active_volunteer_count` to `summary` endpoint
Add the number of active volunteers to the `summary` endpoint. A volunteer counts as active if they completed a submission/made a transcription in the last 2 weeks (or another short time frame). This would give us the opportunity to see how many volunteers are actually still transcribing and whether we are gaining or losing active volunteers.

This can be archived by a query like the following (not tested):
```py
active_volunteer_count = Submission.objects.filter(
    completed_by__isnull=False,
    complete_time__gt=date_minus_two_weeks,
).values("completed_by").count()
```
So we filter the submissions by those that have been completed within the last two weeks, then group it by the volunteers who completed it (so that we have a volunteer count, not a submission count) and then count how many we have.

### PR DESCRIPTION
**Pull request by TimJentzsch**
Relevant issue: Closes #198.

## Description:

This counts the number of volunteers who completed a submission in the last two weeks. This gives us a sense of how many volunteers are actually still transcribing.

## Checklist:

- [x] Code Quality
- [x] Pep-8
- [x] Tests (if applicable)
- [x] Success Criteria Met
- [x] Inline Documentation
- [ ] Wiki Documentation (if applicable)

### CODE REVIEW

File: api/views/misc.py [hunk 1]
```diff
@@ -32,9 +34,17 @@ def generate_summary() -> Dict:
 
         :return: A dictionary containing the three key-value pairs as described
         """
+        date_minus_two_weeks = datetime.datetime.now(tz=pytz.utc) - datetime.timedelta(
+            weeks=2
+        )
         return {
             "volunteer_count": BlossomUser.objects.filter(is_volunteer=True).count()
             - 2,
+            "active_volunteer_count": Submission.objects.filter(
```
Comment (itsthejoker) [inline api/views/misc.py:43]: Will this act as a set and filter out folks who have completed multiples in the last two weeks?

Comment (TimJentzsch) [reply api/views/misc.py:43]: The `.values("completed_by")` essentially groups the submissions by the volunteers. So yes that shouldn't be a problem. If you want I can also add an extra submission to the test cases to verify this behavior.

Comment (itsthejoker) [reply api/views/misc.py:43]: Let's keep the existing test as it is and add one new test for it. Then we'll merge 👍

Comment (TimJentzsch) [reply api/views/misc.py:43]: It indeed didn't work as expected, good thinking!
Should be fixed now though.

**Pull request merged**